docs/csharp/structs.md

```diff
@@ -12,16 +12,64 @@ ms.devlang: csharp
+  
+There are two categories of value types: [struct](https://msdn.microsoft.com/en-us/library/ah19swz4.aspx) and [enum](https://msdn.microsoft.com/en-us/library/sbbt4032.aspx).  
+  
+The built-in numeric types are structs, and they have properties and methods that you can access:  
+  
+[!code-csharp[Static Method](../../samples/snippets/csharp/concepts/structs/static-method.cs)]
+  
+But you declare and assign values to them as if they were simple non-aggregate types:  
+  
+[!code-csharp[Assign Values](../../samples/snippets/csharp/concepts/structs/assign-value.cs)] 
+  
+Value types are *sealed*, which means, for example, that you cannot derive a type from @System.Int32, and you cannot define a struct to inherit from any user-defined class or struct because a struct can only inherit from @System.ValueType. However, a struct can implement one or more interfaces. You can cast a struct type to an interface type; this causes a *boxing* operation to wrap the struct inside a reference type object on the managed heap. Boxing operations occur when you pass a value type to a method that takes a @System.Object as an input parameter. For more information, see [Boxing and Unboxing](https://msdn.microsoft.com/en-us/library/yz2be5wk.aspx).  
+  
+You use the [struct](https://msdn.microsoft.com/en-us/library/ah19swz4.aspx) keyword to create your own custom value types. Typically, a struct is used as a container for a small set of related variables, as shown in the following example:  
+  
+[!code-csharp[Struct Keyword](../../samples/snippets/csharp/concepts/structs/struct-keyword.cs)]  
+  
+For more information about value types in the .NET Framework, see [Common Type System](../standard/common-type-system.md).  
+    
+Structs share most of the same syntax as classes, although structs are more limited than classes:  
+  
+-   Within a struct declaration, fields cannot be initialized unless they are declared as const or static.  
+  
+-   A struct cannot declare a default constructor (a constructor without parameters) or a destructor.  
+  
+-   Structs are copied on assignment. When a struct is assigned to a new variable, all the data is copied, and any modification to the new copy does not change the data for the original copy. This is important to remember when working with collections of value types such as Dictionary<string, myStruct>.  
+  
+-   Structs are value types and classes are reference types.  
+  
+-   Unlike classes, structs can be instantiated without using a **new** operator.  
+  
+-   Structs can declare constructors that have parameters.  
+  
+-   A struct cannot inherit from another struct or class, and it cannot be the base of a class. All structs inherit directly from **System.ValueType**, which inherits from **System.Object**.  
+  
+-   A struct can implement interfaces.  
+  
+-   A struct can be used as a nullable type and can be assigned a null value.
```

Comment:
A nullable corresponding type can be built from any struct type. A struct can't b used as a nullable type.